server/middleware/resetFilterIfRequired.ts

```diff
@@ -0,0 +1,20 @@
+import { RequestHandler } from 'express'
+import logger from '../../logger'
+
+export default function resetFilterIfRequired(): RequestHandler {
+  return async (req, res, next) => {
+    try {
+      if (req.session.journey && req.session.journey.scheduleFilters) {
```

Comment:
This was also previously being carried out in the now removed `populatedCurrentUser.js`, it is still needed.